README.md

```diff
@@ -2,4 +2,40 @@
 
 This repo host an initial code for the tasks performed during the Code Reviewers training.
 
-Application is based on this [Figma Design]()
\ No newline at end of file
+Application is based on this [Figma Design]()
+
+## Primary Task
+- [ ] When the user clicks (or taps) the humburger button, the mobile menu appears
+- [ ] When the user clicks (or taps) the close (X) button, the mobile menu disappears
+- [ ] When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears
+- [ ] When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed
+- [ ] Create a professional README file
+- [ ] Maintain a readable and descriptive commit messages
+
+
+## Set up instrutions
+- [ ] `git clone https://github.com/addod19/mobile-menu.git`
+- [ ] `navigate to the index.html file and double click and if you have live server installed, right click and enjoy the app`
+
+## Linter checks command
+- [ ] `npx stylelint "**/*.scss"`
+- [ ] `npx hint .`
+
+## Mobile Hamburger Screenshot view
+
+![Screenshot from 2021-07-25 22-59-41](https://user-images.githubusercontent.com/46388113/126916119-4d415040-ee10-44de-a806-562dd66fcab9.png)
+
+## Mobile Hamburger Clicked screenshot view
+
+![Screenshot from 2021-07-25 23-00-03](https://user-images.githubusercontent.com/46388113/126916125-3a83960c-fbf2-409a-b886-d72ab2ae5def.png)
+
+
+## Author
+- [ ] [LinkedIn](https://www.linkedin.com/in/daniel-larbi-addo/)
+- [ ] [Stackoverflow](https://stackoverflow.com/users/12172996/daniel-larbi-addo)
+- [ ] [Twitter](https://twitter.com/DanielLarbiAdd1)
```

Comment:
Please use the Microverse template more closely in your readme file because I think the Microverse template is catching and more readable 
![Screenshot from 2021-08-02 13-05-52](https://user-images.githubusercontent.com/62705947/127852319-6d46e99c-beca-4e57-85b3-c3ec77863634.png)

Reply:
Resolved